autotest/cpp/test_osr_set_proj_search_paths.cpp

```diff
@@ -85,6 +85,36 @@ static void func3(void*)
     OSRDestroySpatialReference(hSRS);
 }
 
+static void func4()
```

Comment:
upon reflection, this is going to be very fragile as the database structure might change between PROJ versions. Sorry for not having thought about that before proposing the test. Or maybe build dynamically the aux.db by running a "INSERT INTO geodetic_crs SELECT *, '111111' AS code FROM geodetic_crs WHERE auth_name = 'EPSG' and code = '4326'" (or something like that). The OGR SQLite driver could be used for example. But you'd have to find where proj.db is... Hum. I realize this might become quite complicated and impractical ! Perhaps just test calling the API to check it doesn't crash is the most reasonable option.

Reply:
> upon reflection, this is going to be very fragile as the database structure might change between PROJ versions.

It seems to me that this is not due to changes in the structure of the base. I'm create test_aux.db form original proj.db by deleting everything unnecessary. 

I'm for now add to aux.db one view:
```sql
CREATE VIEW crs_view AS SELECT 'geodetic_crs' AS table_name, auth_name, code, name, type, description, scope, area_of_use_auth_name, area_of_use_code, deprecated FROM geodetic_crs
```
And test on my local build is OK. (Now I'm run all tests at separate branch)

Reply:
> I'm for now add to aux.db one view:

The problem is that the structure of this table has changed the last year (PROJ 7.2 maybe). There is no more scope, area_of_use_auth_name, area_of_use_code in later versions.